src/utils/validations/common.ts

```diff
@@ -76,6 +76,16 @@ export const textField =
     }
   }
 
+export const youtubeVideoLink = (value: string): string => {
+  const youtubeRegex =
+    /^(?:https?:\/\/)?(?:m\.|www\.)?(?:youtu\.be\/|youtube\.com\/(?:embed\/|v\/|watch\?v=|shorts\/|watch\?.+&v=))((\w|-){11})(?:\S+)?$/
```

Comment:
This pattern is not matching all youtube links. For example, the following link is not considered valid, but in reality it has to be valid:
https://youtu.be/watch?v=rhTl_OyehF8&list=RDrhTl_OyehF8&start_radio=1

![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/109922101/214d5cbe-1185-4a28-bbc2-af48217674a3)

I used ChatGpt and it helps to test different shapes of youtube link, you can use it too to be more precise in your regex:

![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/109922101/fc347378-e06c-467a-8989-36f745cf69a5)

Reply:
✅ 
I simplified the pattern so that it only checks for the YT domain name. Now it accepts a wider range of YT links.

Reply:
The regular expression became less complicated, which is great!